packages/themes/src/styles/theme.module.css

```diff
@@ -312,3 +312,42 @@
 .whatsapp-700 {
   background-color: var(--color-whatsapp-700);
 }
+
+.shadowWrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.xs {
+  box-shadow: 0 2px 4px rgba(29, 29, 29, 0.06);
```

Comment:
@esraltintas can you use the shadow variables for these classes, so there is one source of truth. e.g.
```
.s {
  box-shadow: var(--shadow-s);
}
```